### DISCUSSION TITLE
http://localhost:3000/Rocoliilil/14

### DISCUSSION THREAD

**Discussion by giscus** [GDSC-Daejin/gdsc-dju-blog-fe#52] [Announcements]
# http://localhost:3000/Rocoliilil/14

 Google Developers에서 제공하는 프로그램으로, GDSC KR는 21명의 Lead를 주축으로 21개 대학교에 선발되었습니다.

http://localhost:3000/Rocoliilil/14

**Comment by HyeokE** [comment]
또로롱
